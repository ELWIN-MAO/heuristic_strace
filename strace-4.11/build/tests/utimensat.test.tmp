
mym_waitpid=103412

mym_waitpid=103412

mym_waitpid=103412

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=103412

mym_waitpid=103412

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=103412

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103412

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103412

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103412

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103412

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103412

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103412

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103412

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=103412

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=103412

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103412

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103412

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103412

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103412

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103412

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=103412

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=103412

syscall_enter_mym_scno=280  sys_name=utimensat
utimensat(AT_FDCWD, "utimensat\nfilename", [{1489666180, 642432}, {1489666179, 642433}], AT_SYMLINK_NOFOLLOW
mym_waitpid=103412

syscall_exit_mym_scno=280  sys_name=utimensat

mym_scall_rval=0xffffffffffffffff
) = -1 ENOENT (No such file or directory)

mym_waitpid=103412

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103412

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103412

syscall_enter_mym_scno=280  sys_name=utimensat
utimensat(AT_FDCWD, "utimensat\nfilename", [UTIME_NOW, UTIME_OMIT], AT_SYMLINK_NOFOLLOW
mym_waitpid=103412

syscall_exit_mym_scno=280  sys_name=utimensat

mym_scall_rval=0xffffffffffffffff
) = -1 ENOENT (No such file or directory)

mym_waitpid=103412

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=103412

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=103412

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=103412
+++ exited with 0 +++
