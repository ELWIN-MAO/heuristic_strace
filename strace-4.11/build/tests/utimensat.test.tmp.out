utimensat(AT_FDCWD, "utimensat\nfilename", [{1489666180, 642432}, {1489666179, 642433}], AT_SYMLINK_NOFOLLOW) = -1 ENOENT (No such file or directory)
utimensat(AT_FDCWD, "utimensat\nfilename", [UTIME_NOW, UTIME_OMIT], AT_SYMLINK_NOFOLLOW) = -1 ENOENT (No such file or directory)
+++ exited with 0 +++
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
